src/plugins/stylesheet/SassPlugin.ts

```diff
@@ -60,6 +60,7 @@ export class SassPluginClass implements Plugin {
 
         const options = Object.assign({
             data: file.contents,
+            file: context.homeDir+"/"+file.info.fuseBoxPath,
```

Comment:
In this case we need to add it to Post and Less too! It's not just the only plugin.

Reply:
This need to be checked. Don't know if its required. This is a [special behaviour](https://github.com/sass/node-sass#special-behaviours) of node-sass. `file` is only used for sourceMaps if data is provided.